common/views/components/layout/preview.js

```diff
@@ -0,0 +1,17 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
+const styles = readFileSync(join(process.cwd(), '../dist/assets/css/styleguide.css'), 'utf8');
+
+export default (props) => (
+  <html>
+    <head>
+      <style>{styles}</style>
+    </head>
+    <body>
+      <div class="container styleguide__container">
+        <div dangerouslySetInnerHTML={{ __html: props.yield }} />
+      </div>
+    </body>
+  </html>
+);
```

Comment:
What's this?

Reply:
This is the preview layout that wraps each component in Cardigan by default – the equivalent of [this file](https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/views/layout/preview.njk).